frontend/src/rmoods/client/RMoodsClient.ts

```diff
@@ -21,7 +20,7 @@ export class RMoodsClient {
    * @param request
    */
   static async requestReport(request: ReportRequest): Promise<Response> {
-    return await authFetch(`${RMoodsClient.URL}/report`, {
+    return await authFetch(`${BACKEND_URL}/report`, {
```

Comment:
This won't work, the report generation sits at `localhost:8001/api/report`, but you've omitted the `api` part here.